README.md

```diff
@@ -336,16 +336,6 @@ angular
 
 angular
   .module('app')
-  .controller('mainCtrl', [
```

Comment:
Thanks for looking at this! I do not think this is a duplicate avoid block. I was trying to extend the naming conventions that we need to be avoiding `mainCtrl`, `main`, and `MainController`. Instead we should be using `MainCtrl`. Let me know if I missed something and if you need me to clear things up!